src/web/app/components/question-text-with-info/question-text-with-info.component.html

```diff
@@ -1,14 +1,14 @@
 <span class="d-inline-flex align-items-center">
-  <span *ngIf="shouldShowDownloadQuestionResult" class="col-auto">
+  <h3 *ngIf="shouldShowDownloadQuestionResult" class="col-auto question-number">
     <button ngbTooltip="Download Question Results" type="button" class="btn btn-link padding-0" (click)="downloadQuestionResultEvent.emit(); $event.stopPropagation();">
       <b>Question {{ questionNumber }}:</b>
     </button>
-  </span>
-  <span *ngIf="!shouldShowDownloadQuestionResult" class="col-auto">
+  </h3>
+  <h3 *ngIf="!shouldShowDownloadQuestionResult" class="col-auto question-number">
     <b>Question {{ questionNumber }}:&nbsp;</b>
-  </span>
+  </h3>
   <span id="question-text" class="col-auto">&nbsp;{{ questionDetails.questionText }}</span>
-  <a id="additional-info-link" class="info-font-size ms-1" (click)="additionalInfoIsExpanded = !additionalInfoIsExpanded;$event.stopPropagation()" *ngIf="hasAdditionalInfo(questionDetails)" [tmRouterLink]="" queryParamsHandling="merge">
+  <a id="additional-info-link" class="info-font-size ms-1" tabindex="0" (click)="additionalInfoIsExpanded = !additionalInfoIsExpanded;$event.stopPropagation()" *ngIf="hasAdditionalInfo(questionDetails)" [tmRouterLink]="" queryParamsHandling="merge">
```

Comment:
Just realised that semantically this should be a button instead of a link (links take the user to a new location, buttons trigger an action). Shall we try switching this to `<button>` instead?

Currently I am able to select this when using screen reader but not when tabbing, maybe using `<button>` instead would fix this (and we might also not need to add `tabindex="0"`.

Reply:
Good point! Will make the changes!